lib/pipeline.js

```diff
@@ -189,6 +189,37 @@ Pipeline.prototype = {
   },
 
   /**
+   * Insert given streams into the pipeline at a given index
+   * @param {Number} index
+   * @param {...Stream} streams
+   * @returns {Stream[]} inserted
+   */
+  insert: function( index ) {
+
+    if( typeof index !== 'number' ) {
```

Comment:
Yup, sure – I just thought that we can always add it later and `eslint --fix` it – I just wanted to quickly get a proof of concept out, to discuss :)